bin/rb/cap_file_template.rb.erb

```diff
@@ -0,0 +1,9 @@
+set :branch, 'dev' # Which branch you want to deploy from
```

Comment:
It certainly can be. I'll add a property to the site config for which branch to pull from and use that.